pkg/3scale/amp/component/system_searchd.go

```diff
@@ -74,35 +80,21 @@ func (s *SystemSearchd) Deployment(containerImage string) *k8sappsv1.Deployment
 			Template: v1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels:      s.Options.PodTemplateLabels,
-					Annotations: s.Options.PodTemplateAnnotations,
+					Annotations: s.searchdPodAnnotations(watchedSecretAnnotations),
 				},
 				Spec: v1.PodSpec{
+					InitContainers:     s.searchdInit(containerImage),
 					Affinity:           s.Options.Affinity,
 					Tolerations:        s.Options.Tolerations,
 					ServiceAccountName: "amp",
-					Volumes: []v1.Volume{
-						{
-							Name: SystemSearchdDBVolumeName,
-							VolumeSource: v1.VolumeSource{
-								PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
-									ClaimName: SystemSearchdPVCName,
-									ReadOnly:  false,
-								},
-							},
-						},
-					},
+					Volumes:            s.searchdVolume(),
 					Containers: []v1.Container{
 						{
 							Name:            SystemSearchdDeploymentName,
 							Image:           containerImage,
 							ImagePullPolicy: v1.PullIfNotPresent,
-							VolumeMounts: []v1.VolumeMount{
-								{
-									ReadOnly:  false,
-									Name:      SystemSearchdDBVolumeName,
-									MountPath: "/var/lib/searchd",
-								},
-							},
+							//Env:             s.commonSearchdEnvVars(),
```

Comment:
is this required

Reply:
Think it was commented during some debugging . So I would say its required.

Reply:
having reviewed its not required.